modules/apps/frontend-js/frontend-js-test/src/testFunctional/macros/JSONSessionAutomationExtension.macro

```diff
@@ -0,0 +1,24 @@
+definition {
+
+	@summary = "Get a session automation extension status"
+	macro getSessionAutomationExtensionStatus {
+		var baseURL = "${baseURL}";
+
+		if (!(isSet(baseURL))) {
+			var baseURL = PropsUtil.get("portal.url");
+		}
+
+		var curl = '''
+				${baseURL}/c/portal/extend_session \
+				-silent \
+				-o /dev/null \
+				-w '%{http_code}'
+			''';
+		var extendSessionStatus = JSONCurlUtil.get("${curl}");
+
+		echo("extendSessionStatus: ${extendSessionStatus}");
+
+		return "${extendSessionStatus}";
```

Comment:
This value is returned but I don't see where this value is used anywhere else after it's returned?